### PR TITLE
Doc/GitHub handbook

### PR DESCRIPTION
**Pull request by alvarobernal2412**
This PR will resolve the incident #74 .

### CODE REVIEW

**Review by jmartinacu — COMMENTED** [review]
OK

**Review by jmartinacu — APPROVED** [review]

File: doc/Manual-de-Git.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+	- Desde el historial de github de esa rama
+	- Haciendo:
+			`git log --oneline`
+2.  Volver a la rama en la que quieres copiar esos commits
+	1. `git switch <tu_rama>`
+3. Copiar los commits:
+	- Si son muchos:
+			`git cherry-pick <hash_commit_mas_antiguo>^..<hash_commit_mas_nuevo>`
+	- Si son pocos:
+			`git chery pick <hash1> <hash2>`
```
Comment (goncammej) [inline doc/Manual-de-Git.md:29]: There is a typo in this command. It should be 'git cherry-pick ...'

**Review by goncammej — CHANGES_REQUESTED** [review]
Correct the typo, otherwise everything seems to be fine.

**Review by goncammej — APPROVED** [review]
Looks good.

**Pull request merged**